packages/apng2gif.rb

```diff
@@ -3,11 +3,10 @@
 class Apng2gif < Package
   description 'Convert APNG animations into animated GIF format.'
   homepage 'https://sourceforge.net/projects/apng2gif/'
-  @_ver = '1.8'
-  version @_ver
+  version '1.8'
   license 'ZLIB'
   compatibility 'all'
-  source_url "https://sourceforge.net/projects/apng2gif/files/#{@_ver}/apng2gif-#{@_ver}-src.zip"
+  source_url "https://sourceforge.net/projects/apng2gif/files/#{@version}/apng2gif-#{@version}-src.zip"
```

Comment:
`@` not needed?

Reply:
Both `version` and `@version` should be identical, will remove `@` as it is simpler...

Reply:
Done